<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1533</width>
    <height>899</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>561</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>RGB</string>
    </property>
    <widget class="QLabel" name="RGB_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>541</width>
       <height>361</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>561</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>Depth</string>
    </property>
    <widget class="QLabel" name="Depth_label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>461</width>
       <height>351</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>920</width>
      <height>790</height>
     </rect>
    </property>
    <widget class="QFrame" name="mainframe">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>19</y>
       <width>911</width>
       <height>771</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QGroupBox" name="imu_settings_box">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>390</y>
        <width>361</width>
        <height>381</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>IMU Settings</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QCheckBox" name="default_config">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>261</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Use default configuration</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QFrame" name="frame_settings">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>361</width>
         <height>341</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>10</y>
          <width>281</width>
          <height>321</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Number of devices</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="numIMUBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="rate_box">
           <item>
            <property name="text">
             <string>100hz</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>200hz</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>50hz</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>33hz</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>25hz</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="accel_box">
           <item>
            <property name="text">
             <string>4g</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2g</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8g</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16g</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="gyro_box">
           <item>
            <property name="text">
             <string>500dps</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>250dps</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1000dps</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2000dps</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Sample rate</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QComboBox" name="save_box">
           <item>
            <property name="text">
             <string>False</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>True</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Store configuration</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Gyroscope</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Accelerometer</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Algorithm</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="algo_box">
           <item>
            <property name="text">
             <string>kalman</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>compass</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>gyro</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Device</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>False</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>True</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Calibration</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Save calibration</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Load Calibration</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QComboBox" name="calib_box">
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Auto</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Load</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QComboBox" name="comboBox_2">
           <item>
            <property name="text">
             <string>False</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>True</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QGridLayout" name="gridLayout_3" rowminimumheight="0,0">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item row="1" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>shoulder</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="head_device_box">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <item>
              <property name="text">
               <string comment="EXLs3-0239" extracomment="EXLs3-0239">COM7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string comment="EXLs3-0238" extracomment="EXLs3-0238">COM6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>COM3</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="shoulder_device_box">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <item>
              <property name="text">
               <string comment="EXLs3-0239" extracomment="EXLs3-0239">COM6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string comment="EXLs3-0238" extracomment="EXLs3-0238">COM7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>COM3</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>head</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <zorder>gridLayoutWidget</zorder>
       <zorder>gridLayoutWidget_3</zorder>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>240</y>
        <width>361</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>Operation</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>90</y>
         <width>281</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QRadioButton" name="test_radio">
          <property name="text">
           <string>Test</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="save_radio">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="start_btn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="sensor_label">
       <property name="geometry">
        <rect>
         <x>46</x>
         <y>220</y>
         <width>271</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="acquisitions_sources">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>361</width>
        <height>341</height>
       </rect>
      </property>
      <property name="title">
       <string>Acquisition Sources</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>321</width>
         <height>301</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="kinect_radio">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Kinect 2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="imu_radio">
          <property name="text">
           <string>EXLs3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="kinect_imu_radio">
          <property name="text">
           <string>Kinect 2 and IMU</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <zorder>imu_settings_box</zorder>
     <zorder>groupBox_4</zorder>
     <zorder>acquisitions_sources</zorder>
     <zorder>gridLayoutWidget</zorder>
     <zorder>gridLayoutWidget</zorder>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>830</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>FPS: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelFPS">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>830</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QProgressBar" name="calibBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="calibLabel">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>830</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Keep your head as still as possible!</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1533</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHome">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionHome"/>
   </widget>
   <addaction name="menuHome"/>
  </widget>
  <action name="actionHome">
   <property name="text">
    <string>Home</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
